doc/distrib/Samples/en-US/Core/Core_Math.dyn

```diff
@@ -597,39 +597,40 @@ OUT = z</Script>
+    <Dynamo.Models.NoteModel guid="f52a5132-641d-49ee-9178-d7b24f92557d" text="Angles in nodes or Code Block are in radians." x="1836.34388434462" y="-6874.26264410584" />
+    <Dynamo.Models.NoteModel guid="45698910-8d0d-4eb6-a787-7a2310a10234" text="Rounding" x="1854.28521233912" y="-7736.15885661683" />
+    <Dynamo.Models.NoteModel guid="9cb096e6-842d-496b-9e6a-3a871d06bfc4" text="Boolean operators&#xD;&#xA;(Find others in the library.)" x="1839.21824043643" y="-8604.83483561747" />
+    <Dynamo.Models.NoteModel guid="488aa4f1-43a3-4f18-bc19-7a2677f80fb6" text="Tip: Adjust how precisely the geometry in the background preview is rendered.  In the settings menu, choose &quot;Render Precision.&quot;  A higher value will make curved surfaces look smoother, but it will take more processing power to draw." x="2187.04967283522" y="-2263.61659167369" />
+    <Dynamo.Models.NoteModel guid="add548af-ee30-4b0b-9b88-b749caaee89c" text="For more on this example:&#xD;&#xA;http://dynamobim.org/math-transit/" x="2190.02932526022" y="-2399.32775583674" />
+    <Dynamo.Models.NoteModel guid="1c931f1d-3bd7-4cdd-87e3-25dda7135ce2" text="This set of examples shows how to graph a cosine function using nodes, the Formula node, and a Code Block.&#xD;&#xA;&#xD;&#xA;Each example takes a series of numbers as the x-values for a series of points, calculates the y-values from the x-values, and then draws a line in the shape of a cosine curve.  The large code block at the end of each graph moves the drawn geometry to a different part of the screen so that they are not all overlapping." x="1834.17235279297" y="-5384.44101169838" />
+    <Dynamo.Models.NoteModel guid="9eb3e1ec-1e40-4ad9-880a-9e67bad26248" text="Tip: Selecting a node that draws geometry will turn the geometry it creates cyan.  This code block draws the axes of a graph, the points along the curve, and then the curve itself.  Select the code block to see in the background which curve it draws." x="3763.84470264006" y="-4965.35306638978" />
+    <Dynamo.Models.NoteModel guid="49ab4c80-c3ad-4a30-a120-dfc162dca9af" text="Tip: You can turn on or off the preview functionality for any node that makes geometry.  Right click and check or uncheck &quot;Preview.&quot;" x="2681.07141123741" y="-4859.03316263299" />
+    <Dynamo.Models.NoteModel guid="eb29917e-2bca-4394-8aec-5f5a0213fcd1" text="MATH&#xD;&#xA;&#xD;&#xA;Math in Dynamo can be done with nodes from the Math library and the Operators library, with a Formula node, or with a Code Block.  Below, see how to perform several basic math operations with each of these three methods." x="1840.19274183438" y="-9919.72549106897" />
+    <Dynamo.Models.NoteModel guid="91cdaf29-8813-4858-a7c1-cce6d57ea5ed" text="Move the slider to change the length of the arc." x="2198.71039733515" y="-6454.82029082354" />
   </Notes>
+  <Annotations />
 </Workspace>
```

Comment:
@ke-yu That DYN file was having wrong numbering in Note. http://adsk-oss.myjetbrains.com/youtrack/issue/MAGN-7629